docs/build.gradle

```diff
@@ -130,7 +130,6 @@ task generateJavaDocs(type: Javadoc) {
     ext.entryPoint = "$destinationDir/index.html"
 
     if (JavaVersion.current().isJava11Compatible()) {
-        options.addBooleanOption('-no-module-directories', true)
```

Comment:
How about wrapping this in a !JavaVersion.current().isJava12Compatible() so that it's set for java 11 builds, but not 12+.

Reply:
This would also fix #2177

Reply:
That works for me